api/src/main/java/ca/bc/gov/educ/api/batchgraduation/config/BatchJobConfig.java

```diff
@@ -1086,7 +1086,7 @@ public Step schoolReportsRegenJobStep(JobRepository jobRepository, PlatformTrans
 
     @Bean
     public Step masterStepSchoolReportsRegen(JobRepository jobRepository, PlatformTransactionManager transactionManager, EducGradBatchGraduationApiConstants constants, SkipSQLTransactionExceptionsListener skipListener) {
-        int partitionSize = constants.getNumberOfPartitions() / 2;
+        int partitionSize = (constants.getNumberOfPartitions() / 2) + 1;
```

Comment:
Why add one here?

Reply:
Oops, that's my local stuff for debugging.   I will revert this.

Reply:
Done